rnndb/graph/g80_pgraph/tpc.xml

```diff
@@ -42,8 +42,8 @@
 		<bitfield name="DST2D_STORAGE_TYPE_MISMATCH" pos="8" />
 		<bitfield name="ZETA_STORAGE_TYPE_MISMATCH" pos="9" />
 		<bitfield name="RT_STORAGE_TYPE_MISMATCH" pos="10" />
-		<bitfield name="DST2D_LINEAR_MISMATCH" pos="11" />
```

Comment:
ok, since we went with layout elsewhere, this should be LAYOUT_MISMATCH

Hopefully that's all :p